### PR TITLE
Release

### PR DESCRIPTION
**Pull request by NaguiHW**
We did the library project but the TSE said that we have to follow git flow.
https://github.com/NaguiHW/library/pull/3

I hope this time we follow git flow.

### CODE REVIEW

File: library.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    deleteButton.classList.add('btn', 'btn-danger', 'btn-sm', 'delete');
+    deleteBook.appendChild(deleteButton);
+
+    status.addEventListener('click', () => {
+      changeStatus(status);
+    });
+    deleteButton.addEventListener('click', () => {
+      deleteBookFromLocalstorage(deleteButton);
+    });
+  }
```
Comment (JamesOkunlade) [inline library.js:82]: Don't mix the `addBookToLibrary` function with the code that renders the dom. Separate it.

File: library.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+function deleteBookFromLocalstorage(book) {
+  const index = book.parentElement.parentElement.rowIndex;
+  const delBook = JSON.parse(localStorage.books);
+  delBook.splice(index - 1, 1);
+  localStorage.setItem('books', JSON.stringify(delBook));
+  const table = document.querySelector('#book-list');
+  table.deleteRow(index - 1);
+  UI.showAlert('Book Removed', 'info');
+}
```
Comment (JamesOkunlade) [inline library.js]: Use `ES6` arrow function instead.

File: library.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
+class Book {
+  constructor(title, author, pages, read) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.read = read;
+  }
+}
+
+function changeStatus(status) {
```
Comment (JamesOkunlade) [inline library.js]: Use `ES6` arrow function instead.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
### Hello devs ✋ Good job using the `Git flow` 👏 😄 Here are some quick fixes you'll have to do.

- There should be a `addBookToLibrary` function different from a `render` function. i.e. Don't mix the logic of `addBookToLibrary` function with the `render` dom-manipulation code. Your `addBookToLibrary` function should basically use the `Book` constructor to add a book to the book storage array and the `render` function should iterate through the array to display each book in it.
- Change your classic functions to the `ES6 arrow function`.

**Fix those and request for re-review.**

**Status: `Changes required`**

**Happy coding! 😎**

File: library.js (hunk 1 above)
Comment (NaguiHW) [reply library.js:82]: The function `addBookToList` it's already separated from the `addBook`

**Review by AdnanAfsari — APPROVED** [review]
Hello @NaguiHW @bernardogarza ✋ 
Looks good now, Its a 👍 .
**Status: approved**

Happy Coding

**Pull request merged**